go/ct/evm_test.go

```diff
@@ -102,3 +105,246 @@ func TestCt_ExplicitCases(t *testing.T) {
 		})
 	}
 }
+
+const (
+	fuzzIdealStackSize     int = 7  // < max pops in a single instruction
+	fuzzMaximumCodeSegment int = 33 // < 1 instruction with 32 data bytes
+)
+
+// prepareInitialTestData is a helper function to be used by similar fuzzing tests
+// the arguments passed to the f.Add function needs to match the arguments
+// passed to the f.Fuzz function in type, position and number
+func prepareInitialTestData(f *testing.F, rnd *rand.Rand) {
+
+	for revision := R07_Istanbul; revision <= R13_Cancun; revision++ {
+		for op := 0x00; op < 0xFF; op++ {
+			for _, gas := range []int64{0, 1, 6, 10, 1000, math.MaxInt64} {
+
+				// generate a code segment with the operation followed by 6 random values
+				ops := [fuzzMaximumCodeSegment]byte{}
```

Comment:
You should probably prefer slices over arrays here and for the `stack` variable below.